### PR TITLE
convert string true to bool so tinymce can understand

### PR DESCRIPTION
**Pull request by mvanhelmont**
Converting string true from tinyMceConfig.config customConfig to a bool
object. so remove_script_host can be used.

### CODE REVIEW

File: src/Umbraco.Web/Models/ContentEditing/RichTextEditorConfiguration.cs [hunk 1]
```diff
@@ -23,6 +23,6 @@ public class RichTextEditorConfiguration
         public string InvalidElements { get; set; }
 
         [DataMember(Name = "customConfig")]
-        public IDictionary<string,string> CustomConfig { get; set; }
+        public IDictionary<string,object> CustomConfig { get; set; }
```
Comment (Shazwazza) [inline src/Umbraco.Web/Models/ContentEditing/RichTextEditorConfiguration.cs]: This would be a breaking change, it will need to stay as `string`

**Comment by Shazwazza** [comment]
Can you please create an issue on the tracker with details about what this solves so we can track it in a release?

**Comment by mvanhelmont** [comment]
The issue is U4-8724. The intention for me was to fix the absolute url problem in tinymce. When you set a custom config convert_urls and remove_script_host to false normally this problem goes away. But because the false value is serialized to a string tinymce does not understand him.

I have seen in de current release 7.5.3 that convert_urls and remove_script_host are hardcoded in the RTEController and set to false. So for me the problem is already solved.

**Comment by Shazwazza** [comment]
Could you point me in the direction of the code where convert_urls and remove_script_host are hard coded now? ( can't seem to find this )

**Comment by mvanhelmont** [comment]
Oh sorry i really thought i have seen it in the source of rte.controller.js but i have fixt it for one of our sites that way.

**Comment by Shazwazza** [comment]
I think remove_script_host is there but no the other one, i'll see if we can review this more thoroughly next sprint

**Comment by nul800sebastiaan** [comment]
Hi @mvanhelmont - very sorry for the super late reply!

Unfortunately, as Shannon pointed out here, changing from string to object would be a breaking change. Turns out the change was pretty easy to do in javascript instead. So, inspired by your idea, I made a few tweaks to the Javascript and true/false options will (finally) be available as of 7.10!
https://github.com/umbraco/Umbraco-CMS/commit/e3e425695f724cbbf1b25e2dfe97d90d8cdcd9f2

Thanks very much for your efforts and again sorry for the late reply.

**Comment by ronaldbarendse** [comment]
This change should be merged to dev-v8, together with any changes in the AngularJS RTE controller/grid RTE directive.

**Pull request closed without merging**